### PR TITLE
VS Code first project edits

### PR DESCRIPTION
**Pull request by j-c-berger**
Signed-off-by: Jacob Berger <jacob.berger@ibm.com>

For issue 1756: https://github.com/eclipse/codewind/issues/1756

I simply modified the content to fit our restructure 

@micgibso or @sishida, please review and either let me know what additional corrections must be made or merge. 

Thanks-JB

### CODE REVIEW

File: docs/_documentations/mdt-vsc-firstproject.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: docs
-title: "Codewind for VS Code"
-description: "Codewind for VS Code"
+title: "Creating your first project"
+description: "Creating your first project"
```
Comment (sishida) [inline docs/_documentations/mdt-vsc-firstproject.md]: If a user comes to this page directly, such as through an internet search, they might not be aware of the table of contents and how this topic fits in with the bigger picture. Also, we might have a page for "Creating your first project" for each IDE, right?

If these are the cases, then I think we should mention the IDE as part of the title. Maybe something like "Creating your first Codewind project with VS Code" or "Creating your first VS Code Codewind project"?

File: docs/_documentations/mdt-vsc-firstproject.md [hunk 2]
```diff
@@ -15,23 +15,23 @@ To create your first project, click on the prompt in the **CODEWIND** window
...
 
 ![](dist/images/nodetemplate.png)
 
-Selecting this template will prompt you to enter a name for your project
+Selecting this template prompts you to enter a name for your project
 
 ![](dist/images/projectname.png)
 
-Enter a name and the choose where to create this on disk (for example myFirstNodeProject)
+Enter a name and then choose where to create this on disk. For example, myFirstNodeProject
```
Comment (sishida) [inline docs/_documentations/mdt-vsc-firstproject.md]: Lines 18, 26, and 30 could all have periods at the end...or maybe a colon instead for lines 18 and 26. Also, "myFirstNodeProject" could be in `code phrase format`.

File: docs/_documentations/mdt-vsc-firstproject.md [hunk 3]
```diff
@@ -42,11 +42,11 @@ To view your running application, click on the project name in the **CODEWIND**
 and select the icon
 ![](dist/images/launchicon.png)
 
-This will launch your web browser and display your application
+This icon launches your web browser and displays your application
```
Comment (sishida) [inline docs/_documentations/mdt-vsc-firstproject.md]: This line could also end with a period or colon.

**Comment by j-c-berger** [comment]
@sishida, thank you for the review. I applied the changes. You or @micgibso can review and/or merge, unless there's more to do!

**Comment by j-c-berger** [comment]
@micgibso, new commit is ready for review and hopefully merging. I already pinged Sarah about it. Thanks!

File: docs/_documentations/mdt-vsc-firstproject.md [hunk 4]
```diff
@@ -1,52 +1,54 @@
...
 <br/>
-To create your first project, click on the prompt in the **CODEWIND** window
+To create your first project:
+
+1\. Click on the prompt in the **CODEWIND** window.
 
 ![](dist/images/createproject.png)
 
-This will prompt you with a list of project types to select from.
+2\. This prompt gives you a list of project types for you to select.
```
Comment (micgibso) [inline docs/_documentations/mdt-vsc-firstproject.md]: Not keen on `gives`, how about: 
This prompt displays a list of project types. Select the project type that you require from the list.

**Pull request merged**